draft-ietf-acme-acme.md

```diff
@@ -1699,13 +1763,25 @@ Content-Type: application/jose+json
 
 The CSR encodes the client's requests with regard to the content of the
 certificate to be issued.  The CSR MUST indicate the exact same set of requested
-identifiers as the initial new-order request, either in the commonName portion
+identifiers as the initial new-order request.  Identifiers of type "dns" MUST apper in either in the commonName portion
 of the requested subject name, or in an extensionRequest attribute {{!RFC2985}}
-requesting a subjectAltName extension.
+requesting a subjectAltName extension.  (These identifiers may appear
+in any order.)  Specifications that define
```

Comment:
fwiw: I tried to always say "sort order" when talking about this to avoid conflating with ACME Orders. I think it might help clarify to do the same here.

Reply:
Ah, English...